velocity-transition-group.js

```diff
@@ -44,6 +44,13 @@ var ReactDOM = require('react-dom');
 var ReactTransitionGroup = require('react-addons-transition-group');
 var Velocity = require('./lib/velocity-animate-shim');
 
+// Shim requestAnimationFrame for browsers that don't support it, in particular IE 9.
+var shimRequestAnimationFrame = 
```

Comment:
Sorry, just noticed that you updated this. The check has to be `typeof window !== 'undefined'` since just referencing `window` as-is leads to a `ReferenceError`.

Unfortunately we do not have test coverage for the server-side bits, or else this would be easier to sort out.